hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/metadata/SCMDBDefinition.java

```diff
@@ -75,6 +80,15 @@
           X509Certificate.class,
           new X509CertificateCodec());
 
+  public static final DBColumnFamilyDefinition<BigInteger, CertInfo>
+      REVOKED_CERTS_V2 =
```

Comment:
@bharatviswa504 Is `REVOKED_CERTS_V2` needed to be replicated across multi SCMs if SCM HA Is on?

Reply:
Yes, it should be replicated across SCM's.

Reply:
Thanks for bringing this up @GlenGeng and @bharatviswa504. I have created a separate jira to fix the persistence via Ratis - https://issues.apache.org/jira/browse/HDDS-5095 since we also need to fix the valid certs table.

Reply:
It seems like a small change. So, I will update the same patch to get "REVOKED_CERTS_V2" replicated.

Reply:
@vivekratnavel  and I have an offline discussion on this. Both crl sequence id and revoked certs replication will be taken care by HDDS-5095 via Replicate annotation on the CertificateStore#revokeCertificats